src/targets/javascript/fetch.js

```diff
@@ -21,30 +23,39 @@ module.exports = function (source, options) {
     options
   )
 
-  const stringifyObject = require('stringify-object')
   const code = new CodeBuilder(opts.indent)
 
-  options = {
```

Comment:
I renamed this variable to `reqOpts` so it's not overwriting the `options` argument of the target.